### DISCUSSION TITLE
Version compatibility with Flet-FastApi

### DISCUSSION THREAD

**Discussion by ClearSafety** [flet-dev/flet#3745] [Q&A]
### Question

I'm converting my app into a version using flet-fastapi. I got an error, it said page has not the window property. For some reason, after installing flet-fastapi Flet turned to the version 0.19.0.
After upgrading it, I got an error saying flet-fastapi requires flet version 0.19.0.
Am I doing anything wrong or indeed I'll have to use Flet version 0.19.0 if I wanna use flet-fastapi?

### Error message

```bash
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
flet-fastapi 0.19.0 requires flet==0.19.0, but you have flet 0.23.1 which is incompatible.
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Read [this](https://medium.com/python-in-plain-english/whats-new-in-flet-0-21-0-ca482ab4520b#:~:text=package%20is%20now-,deprecated,-and%20its%20contents).
